k8s.io/configmap-nginx.yaml

```diff
@@ -158,7 +158,7 @@ data:
           # Don't require /release/ if you want to get at the Kubernetes release artifacts, the common case.
           rewrite ^/(v[0-9]+\.[0-9]+\.[0-9]+(-(alpha|beta|rc)\.[0-9]+)?/.*)$   https://storage.googleapis.com/kubernetes-release/release/$1 redirect;
           # Provide a convenient redirect for CI (continuous integration) artifacts as well, which live in a different bucket.
-          rewrite ^/ci(-cross)?/?(.*)$              https://storage.googleapis.com/kubernetes-release-dev/ci$1/$2 redirect;
+          rewrite ^/ci/?(.*)$              https://storage.googleapis.com/k8s-release-dev/ci$1/$2 redirect;
```

Comment:
Two things I noticed when trying to deploy:
- `ci-cross` is still referenced here, and cutting .debs may be broken until this code is fixed: https://github.com/kubernetes/release/blob/6e3b30b6dd78b705474a1086f2ed7f7b8364029b/packages/deb/build.go#L291-L302
- remove the first capture group `(-cross)` means the the replacement is incorrect

Fixup PR inbound

Reply:
Ahhh, regex, my old enemy.
Thanks for the cleanup, Aaron.

I'll fixup the k/release references.

Reply:
Opened https://github.com/kubernetes/k8s.io/pull/2292

Reply:
>     * `ci-cross` is still referenced here, and cutting .debs may be broken until this code is fixed: https://github.com/kubernetes/release/blob/6e3b30b6dd78b705474a1086f2ed7f7b8364029b/packages/deb/build.go#L291-L302

Fixed in https://github.com/kubernetes/release/pull/2153.